### DISCUSSION TITLE
Serialization to ISO 8601

### DISCUSSION THREAD

**Discussion by lungu-catalin** [laravel/framework#42924] [General]
Hi all,

I'm migrating a booking application from Laravel 6 to 7 and I realized that the default format for the date is ISO 8601. This happen only for timestamp fields  or for all date types?
For example:
TIMESTAMP field -> MyModel::first()->created_at returns "2020-02-18T07:42:08.000000Z"
DATETIME field -> MyModel::first()->field_1 returns "2020-12-25 00:00:00"
DATE field -> MyModel::first()->field_2 returns "2020-12-25 00:00:00"
TIME field -> MyModel::first()->field_3 returns "00:00:00"

Also only created_at allow toJson() method. So is my assumption right?

**Comment by bonzai** [comment]
https://laravel.com/docs/7.x/upgrade#date-serialization
